index.html

```diff
@@ -49,6 +49,232 @@ <h2>I’am a Software Developer</h2>
+                            been the industry's standard
+                        </p>
+                        <ul class="projects_stack">
+                            <li class="projects_stack_item projects_stack_item1"><span>html</span></li>
+                            <li class="projects_stack_item projects_stack_item2"><span>bootstrap</span></li>
+                            <li class="projects_stack_item projects_stack_item3"><span>Ruby</span></li>
+                        </ul>
+                    </div>
+                    <div class="action">
+                        <button type="button" class="see_project_action_btn"><span class="btn_text">See Project</span></button>
+                    </div>
+                </div>
+                <div class="other_works_item5 other_works_item">
+                    <div class="right_block">
+                        <h3>Profesional Art Printing Data</h3>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has
+                            been the industry's standard
+                        </p>
+                        <ul class="projects_stack">
+                            <li class="projects_stack_item projects_stack_item1"><span>html</span></li>
+                            <li class="projects_stack_item projects_stack_item2"><span>bootstrap</span></li>
+                            <li class="projects_stack_item projects_stack_item3"><span>Ruby</span></li>
+                        </ul>
+                    </div>
+                    <div class="action">
+                        <button type="button" class="see_project_action_btn"><span class="btn_text">See Project</span></button>
+                    </div>
+                </div>
+                <div class="other_works_item6 other_works_item">
+                    <div class="right_block">
+                        <h3>Profesional Art Printing Data</h3>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has
+                            been the industry's standard
+                        </p>
+                        <ul class="projects_stack">
+                            <li class="projects_stack_item projects_stack_item1"><span>html</span></li>
+                            <li class="projects_stack_item projects_stack_item2"><span>bootstrap</span></li>
+                            <li class="projects_stack_item projects_stack_item3"><span>Ruby</span></li>
+                        </ul>
+                    </div>
+                    <div class="action">
+                        <button type="button" class="see_project_action_btn"><span class="btn_text">See Project</span></button>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="about_block">
```

Comment:
Kindly change this from a `div` tag to a `section ` tag. This is a stand alone section on its own and using semantic elements is important so that screen readers understand your project.